pkg/bootkube/create.go

```diff
@@ -44,6 +46,9 @@ func CreateAssets(manifestDir string, timeout time.Duration) error {
 	if err := wait.Poll(5*time.Second, timeout, createFn); err != nil {
 		return fmt.Errorf("Failed to create assets: %v", err)
 	}
+
+	fmt.Println("Spawning self-hosted control plane...")
```

Comment:
Hmm, I see your point. We could put this message in the AllRunning() func right where we print the actual pod status, but we need a persistent bool var to guard printing it only once. So it gets a little ugly either way since messaging about asset creation vs pod status will always be racey and it seems weird to synchonize them just for the user messages. I do want to leave the message in. Have a preference either way?